### ISSUE TITLE
PHP Warning in App.php

### ISSUE THREAD

**Issue by miqrogroove** [friendica/friendica#5264]
Something weird going on with setPConfigValue() so I'm guessing this was caused somewhere else in the code.

[20-Jun-2018 15:38:38 UTC] PHP Warning:  Illegal string offset 'css_modified' in /src/App.php on line 1049

**Comment by MrPetovan** [comment]
This error happens when we try to access a string variable with the `[]` operator. For example:
```php
$string = 'test';
echo $string[1]; // 'e'
echo $string['css_modified']; // PHP Warning: Illegal string offset 'css_modified'
```

The question is why `$this->config[$uid][$cat]` is a string in either of those calls:
https://github.com/friendica/friendica/blob/e2b4690d6f1f3831c6d282e7ad5b89900c7b9b94/view/theme/frio/style.php#L35
https://github.com/friendica/friendica/blob/e2b4690d6f1f3831c6d282e7ad5b89900c7b9b94/view/theme/frio/config.php#L26

**Comment by miqrogroove** [comment]
https://github.com/friendica/friendica/blob/b6b67c9044b22a2f04d17dd03a81c794919a27a2/src/Core/Config/JITPConfigAdapter.php#L36

**Comment by MrPetovan** [comment]
Ha, good lead, I'm no using the JIT config adapter anymore so I didn't think about looking over there.

**Comment by miqrogroove** [comment]
I don't even know what it does.  I just found that by searching 😁

**Comment by MrPetovan** [comment]
Fixed in #5526 in https://github.com/friendica/friendica/pull/5526/commits/dd0fc38c8e74a6d1469da614006caae623974540
